pkg/cmd/cli/cmd/set/env.go

```diff
@@ -200,8 +202,23 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 			return nil
 		})
 		if !ok {
-			skipped++
-			continue
+			ok, err = f.UpdateObjectEnvironment(info.Object, func(vars *[]kapi.EnvVar) error {
```

Comment:
The goal is to move set env upstream, so we will need it.

On Thu, Feb 25, 2016 at 12:12 PM, Michail Kargakis <notifications@github.com
> wrote:

> In pkg/cmd/cli/cmd/set/env.go
> <https://github.com/openshift/origin/pull/7625#discussion_r54138076>:
>
> > @@ -200,8 +202,23 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
> >  			return nil
> >  		})
> >  		if !ok {
> > -			skipped++
> > -			continue
> > +			ok, err = f.UpdateObjectEnvironment(info.Object, func(vars *[]kapi.EnvVar) error {
>
> I guess I am fine with two separate methods. @smarterclayton
> <https://github.com/smarterclayton> is there any use for this upstream?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7625/files#r54138076>.
>